src/Documentation/SidebarMenu/SidebarMenu.js

```diff
@@ -2,9 +2,6 @@ import React, { Fragment } from 'react'
 import $ from 'jquery'
 // components
 import DownloadButton from '../../DownloadButton'
-// utils
-import startCase from 'lodash.startcase'
-import includes from 'lodash.includes'
```

Comment:
do we use lodash anywhere else? can we remove it from the requirements?

Reply:
they were not useful this